src/utils/ecommerce/auth-request.ts

```diff
@@ -12,7 +15,9 @@ export const createSuccessAuthRequestResponseEntityFromNPG = (
   jsonResponse: any
 ): RequestAuthorizationResponse => ({
   authorizationRequestId: "requestId",
-  authorizationUrl: jsonResponse.url
+  authorizationUrl:
+    "http://localhost:1234/gdi-check#gdiIframeUrl=" +
+    encode(jsonResponse.fieldSet.fields[0].src)
```

Comment:
Here we are assuming that Front End is always available on port 1234. Maybe extracting the port info from the request and using that data is a better choice

Reply:
Did it!